### PR TITLE
Copy core directory from signify ts

### PR DESCRIPTION
**Pull request by jrayback**
Recreation of files from `signify-ts/core` as literally as possible. Rudimentary `npm start` capability. Includes all tests from `signify-ts/test/core` passing. `npm run build` works.

### CODE REVIEW

**Comment by jrayback** [comment]
This is as literal as I could get and have things still work. I stuck with all dependencies as is - but strongly recommend we audit them and update as warranted - some are _very_ sketchy. All tests pass (albeit slowly at ~12s) using Jest directly. We've already discussed moving to the native test runner, so this is probably a temporary configuration. Admittedly, the directory structure is a bit contorted due to some dependencies being awkwardly placed in `signify-ts` - I've isolated these two or three exports by commenting out everything around them - seems totally doable to reorganize these more reasonably for the form factor of `cesr-ts`. There's no prettifier or linting yet, will tackle that in a new PR. Also I expect there are some files pulled over from `signify-ts` that aren't needed here (holder functionality, e.g.)...so please identify stuff I should delete in additional commits before merging this PR.

File: src/authing.ts
Comment (m00sey) [inline src/authing.ts:1]: You don't need this file.

File: src/end/ending.ts
Comment (m00sey) [inline src/end/ending.ts:1]: Don't need this file

File: src/httping.ts
Comment (m00sey) [inline src/httping.ts:1]: or this

File: src/eventing.ts
Comment (m00sey) [inline src/eventing.ts:1]: or this

**Review by m00sey — COMMENTED** [review]
don't think you need manager or keeping either, when I was trying to do a "signified" I found they were better suited to the signify layer vs the vest one

File: src/authing.ts
Comment (jrayback) [reply src/authing.ts:1]: Done.

File: src/end/ending.ts
Comment (jrayback) [reply src/end/ending.ts:1]: done.

File: src/httping.ts
Comment (jrayback) [reply src/httping.ts:1]: done

File: src/eventing.ts
Comment (jrayback) [reply src/eventing.ts:1]: deleted vdring too -> was dependent on eventing. seemed not to be core CESR functionality.

**Comment by jrayback** [comment]
OK, I deleted unnecessary files. Please see itemized responses to comments above. Tests pass and code compiles. I think this is ready to merge. Further cleanup can happen as we go through the process of developing the desired api to make this a proper npm package.

**Review by m00sey — APPROVED** [review]
lgtm

**Pull request merged**